site/docs/guide/deployment-types.md

```diff
@@ -56,9 +56,9 @@ ______________                                   _____________
 ```
 
 It is immediately obvious that this has some drawbacks.
-Your bot only receives new messages every time it asks, i.e. every few seconds or so.
-To make your bot respond faster, you could just send more requests and not wait as long between them.
-We could for example ask for new messages every millisecond! What could go wrong…
+Your bot only receives new messages every time it asks, i.e., every few seconds or so.
+To make your bot respond faster, you could just send more requests and not wait too long between them.
```

Comment:
This is false. We are comparing. It's not about some absolute time interval that should _not be too long_. Please revert.

Reply:
I will go through all and revert them all if needed.

Reply:
This exact case only occurs once in your suggestions.